extensions/wikia/TemplateClassification/templates/TemplateClassificationViewPageEntryPoint.mustache

```diff
@@ -1,3 +1,5 @@
 {{templateTypeLabel}}
-<span class="template-classification-type-text" data-type="{{templateType}}">{{templateTypeName}}</span>
-{{#editButton}}<a class="template-classification-edit sprite-small edit"></a>{{/editButton}}
+{{#editButton}}<a href="#" class="template-classification-edit" data-type="{{templateType}}">
+	{{templateTypeName}}<span class="template-classification-edit sprite-small edit"></span>
```

Comment:
Please remove `template-classification-edit` class from `span` to avoid loading and opening modal twice which cause js error (because we bind this event to this class)